### DISCUSSION TITLE
A possible option to set 'Open in New Tab' as the default action?

### DISCUSSION THREAD

**Discussion by Aquarius-62** [wilgaboury/super-speedy-dial#79] [Ideas]
Greetings.

I am used to (with other speed dials) just clicking on the icon/dial and having it open in a new tab.

Would it be possible, or is it something that can easily be edited on my end, to have an option for Open in New Tab to be the default action? Not a deal breaker, just wondering.

Have a great week :)
Fritz

**Comment by wilgaboury** [comment]
Hi, thanks for creating a discussion about this. Conceptually, each tile works like a link, click opens it in the current tab, ctrl+click will open the bookmark in a new tab. In order to keep the plugin intuitive and inline with web conventions, I would rather not change the default click behavior.

If you are looking to make just click open in a new tab, there is not currently a way to do that without forking the repository, modifying the code and then building and installing the plugin manually.

I can make a ticket for adding settings for changing the default behavior in the settings side-panel, but just be understanding that I have a lot of other features+improvements currently planned that will take priority over it.

**Reply by Aquarius-62** [reply]
Hi, and thanks for the reply. I understand, no problem. I just wanted to throw it out there and see.
Have a great week :)
